package.json

```diff
@@ -41,6 +41,6 @@
   "license": "",
   "name": "go-libp2p-raft",
   "releaseCmd": "git commit -a -m \"gx publish $VERSION\"",
-  "version": "0.0.2"
+  "version": "0.1.0"
```

Comment:
yep yep it should have been a patch-level bump. also it should probably not bump and release top-level packages, like go-libp2p-raft currently is since nothing else depends on it.

nevermind! :)